### ISSUE TITLE
[Panel] Cannot Delete Page With Subpages (3.2.0-rc.2 Regression)

### ISSUE THREAD

**Issue by neildaniels** [getkirby/kirby#1851]
**Describe the bug**
When trying to delete a page with subpages, the Panel dialog does not actually let me delete the page.

**To Reproduce**
1. Go to a page with subpages
2. Settings > Delete
3. Type in page title
4. Click "Delete"

**Expected behavior**
Page should be deleted. Instead, clicking the Delete button does nothing. 

**Screenshots**
<img width="511" alt="Screen Shot 2019-06-11 at 10 22 20 AM" src="https://user-images.githubusercontent.com/417182/59293219-9acf2c00-8c33-11e9-818c-78cab73e17d4.png">

**Kirby Version**
`3.2.0-rc.2`

**Desktop (please complete the following information):**
 - OS: macOS 10.14.5 (18F132)
 - Safari 12.1.1 (14607.2.6.1.1)

**Additional context**
Deleting pages without subpages works fine.

**Comment by distantnative** [comment]
Anyything in the console after hitting the delete button?

**Comment by neildaniels** [comment]
@distantnative Nope, nothing showing up in the Console for me.

This seems to _only_ affect Safari. Deleting pages with subpages works fine in Chrome (75.0.3770.80).

**Comment by neildaniels** [comment]
There's nothing in the console for me.

This may have been a file-permissions issue on my test server and local instance.

When I tried in Chrome, it gave me the following error:
> rename(): The first argument to copy() function cannot be a directory

Then Safari started giving me that error as well. (Not sure why it didn't before.)

I changed some file permissions and now both browsers work fine.

Unless you suspect something has changed around this area, I think this should be safe to close.
